pkg/config/config.go

```diff
@@ -826,7 +826,6 @@ func trimTrailingSlashFromURLS(config Config) error {
 	var additionalEndpointSelectors = []string{
 		"additional_endpoints",
 		"apm_config.additional_endpoints",
-		"logs_config.additional_endpoints",
```

Comment:
No more removing trailing slash ?

Reply:
Logs_config.additional_endpoints uses a different structure than other addtional_endpoints (cf. https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/config/endpoints.go#L13-L21),  and this loop was overwriting the config key logs_config.additional_endpoints with something inconsistent).
And it uses host & port fields so no need for trailing slash removal.